examples/05_deduplication.py

```diff
@@ -131,48 +118,71 @@
 )
 deduplicated_series = pd.Series(deduplicated_counts, index=deduplicated_unique_examples)
 
-deduplicated_series.plot.barh(figsize=(10, 15))
-plt.xlabel("Medication name")
-plt.ylabel("Counts")
+deduplicated_series.plot.barh(figsize=(10, 5))
```

Comment:
having to create a pandas series just for plotting seems a bit weird, why not having:
```python
plt.barh(deduplicated_counts, deduplicated_unique_examples)
```
?